ports/unix/variants/coverage/mpconfigvariant.mk

```diff
@@ -26,7 +26,6 @@ CFLAGS += -DCIRCUITPY_QRIO=1
 $(BUILD)/lib/quirc/lib/%.o: CFLAGS += -Wno-shadow -Wno-sign-compare -include shared-module/qrio/quirc_alloc.h
 
 SRC_BITMAP := \
-	$(patsubst ../../%,%,$(wildcard ../../shared-bindings/gifio/*.c ../../shared-module/gifio/*.c)) \
```

Comment:
Just a comment on this: I removed it as the unix port was not building with `OnDiskGif` compiled in now as part of `gifio`. The only test I saw using this was the one on which modules were built in. If I'm missing something please let me know.

Reply:
@jepler may have added this when testing the gif writing

Reply:
I tested gifio in the unix port when implementing gif writing. However, there are not currently any automated tests that use it (e.g., to verify that gifio still operates properly) so there's no "real regression" (such as reduced testing) due to disabling this.